lib/matplotlib/backends/backend_ps.py

```diff
@@ -1652,55 +1638,48 @@ def pstoeps(tmpfile, bbox=None, rotated=False):
         bbox_info, rotate = None, None
 
     epsfile = tmpfile + '.eps'
-    with io.open(epsfile, 'wb') as epsh:
+    with io.open(epsfile, 'wb') as epsh, io.open(tmpfile, 'rb') as tmph:
```

Comment:
I guess...
On the other hand there seems to be an awful lot use of temporary files in backend_ps, most of which could probably just get rewritten using pipes to communicate via the subprocesses' standard streams.
Another great cleanup project :-)